### PR TITLE
Update Angular Spanish tutorial to the English version

### PR DESCRIPTION
**Pull request by josegarciamanez**
Here is my input for the Update.
@jonniebigodes

### CODE REVIEW

**Comment by jonniebigodes** [comment]
@josegarciamanez this is so great! Thank you very much for the translation. If you're ok with it I'm going to close the other pull request in favor of this one, more specifically #473, and get a proofreader to check the pull request.

Sounds good?

**Comment by josegarciamanez** [comment]
OK @jonniebigodes

**Comment by jonniebigodes** [comment]
@josegarciamanez just a friendly heads up regarding the translation status? Are still able to work on it. Don't forget that the Angular translation was moved into Storybook 6.2 with #482.

Let me know and we'll adjust accordingly.

Stay safe

**Comment by josegarciamanez** [comment]
@jonniebigodes Sorry, I	ve had a lot of work this month. This week I will be on vacation and will work on the update to version 6. 2 to send the pull request. 


All the best

**Comment by josegarciamanez** [comment]
@jonniebigodes Would you be interested in me also working on the translation of the design system documentation into Spanish? Stay tuned. 


All the best

**Comment by josegarciamanez** [comment]
@jonniebigodes Everything would already be updated. 

Stay safe

**Comment by jonniebigodes** [comment]
@josegarciamanez no worries, glad to hear that you're willing to work on this while you're on vacation. Really appreciate it 🙏🏼  . I'm going to take a look during this week and if anything stands out we'll get it proofread and hopefully merged.

Stay safe

File: content/intro-to-storybook/angular/es/composite-component.md [hunk 1]
```diff
@@ -224,50 +227,52 @@ A medida que el componente crece, también lo hacen los parámetros de entrada r
...
 
-En el capítulo anterior aprendimos a capturar historias de prueba utilizando Storyshots. Con el componente `TaskComponent` no había mucha complejidad para probar más allá de que se renderice correctamente. Dado que `TaskListComponent` añade otra capa de complejidad, queremos verificar que ciertas entradas produzcan ciertas salidas de una manera adecuada con pruebas automáticas. Para hacer esto crearemos test unitarios utilizando [Jest](https://facebook.github.io/jest/) junto con un renderizador de prueba.
+En el capítulo anterior aprendimos a capturar historias de prueba utilizando Storyshots. Con el componente `TaskComponent` no había mucha complejidad para probar más allá de que se renderice correctamente. Dado que `TaskListComponent` añade otra capa de complejidad, queremos verificar que ciertas entradas produzcan ciertas salidas de una manera adecuada con pruebas automáticas. Para hacer esto crearemos test unitarios utilizando [Angular Testing Library](https://testing-library.com/docs/angular-testing-library/intro).
 
 ![Jest logo](/intro-to-storybook/logo-jest.png)
 
-### Test unitarios con Jest
+<div class="aside">
+TODO:ADD testing library image
+</div>
```
Comment (jonniebigodes) [inline content/intro-to-storybook/angular/es/composite-component.md:236]: @josegarciamanez can you check this. It seems that this div should not be present.

File: content/intro-to-storybook/angular/es/get-started.md [hunk 2]
```diff
@@ -1,56 +1,79 @@
...
+# Clonar la plantilla
+npx degit chromaui/intro-storybook-angular-template taskbox
+
 cd taskbox
 
-# Añade Storybook:
-npx -p @storybook/cli sb init
+# Instalar dependencias
+npm install
 ```
```
Comment (jonniebigodes) [inline content/intro-to-storybook/angular/es/get-started.md:33]: @josegarciamanez If you could leave the code comments in their native form (English) throughout the tutorial,we'd appreciate it. I'm aware that translating them adds context and brings the translation closer to home but until we have a good solution for this we recommend that they remain as mentioned.

File: content/intro-to-storybook/angular/es/simple-component.md [hunk 3]
```diff
@@ -1,378 +1,381 @@
...
+```diff:title=.storybook/preview.js
+import { setCompodocJson } from "@storybook/addon-docs/angular";
+import docJson from "../documentation.json";
+setCompodocJson(docJson);
+
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
++ angularLegacyRendering: true,
+}
```
Comment (jonniebigodes) [inline content/intro-to-storybook/angular/es/simple-component.md:204]: @josegarciamanez if you could adjust this portion to what's currently in the English version we'd appreciate it. I pushed a small pull request #488 that addressed this. Long story short this was introduced as a temporary solution that I was able to sort out with the current Storybook Angular maintainer and with his guidance I updated the tutorial accordingly.

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@josegarciamanez this is shaping up real nice and we're almost in a good place to start proofreading it. I'm going to take a closer look once you address the items I've mentioned and I'll ask a team member as well for a sanity check.

Let me know once you've addressed them and we'll go from there.

Stay safe

**Comment by josegarciamanez** [comment]
@jonniebigodes  In my last commit you will find the required changes. All the best

**Comment by jonniebigodes** [comment]
@josegarciamanez if you don't mind factoring in #504 in this pull request we'd appreciate it. I've taken a look at it and it seems that is good to go. I'm going to ask for an "extra pair of eyes" if you don't mind to see if I missed something and we'll gladly merge it sometime next week. Sounds good?

**Review by amartinez62 — APPROVED** [review]
LGTM🎉

**Comment by jonniebigodes** [comment]
@josegarciamanez sorry for the long wait on this. I got held up and had to take care of some other aspects of the documentation. I'm going to merge this.

**Pull request merged**